cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -204,6 +219,14 @@ def setUpClass(cls):
         cls.VerifyingError = -2
         cls.UpdatingError = -3
 
+    @property
+    def good_file(self):
+        return random.choice([self.good_tar, self.good_zip])
```

Comment:
The idea here is to let CI test each time in different way, like what we do creating [factory_boy](https://factoryboy.readthedocs.io/en/stable/) and faker libraries

Reply:
@rodfersou, what's the point of this? These are four different cases, so we should run them all every time instead of randomly deciding which one will run. If one of these cases fails, we won't even know which one it was.

We should specify all cases with `@ddt.data`.